_includes/includes/ui/keyboard-details.php

```diff
@@ -55,7 +55,7 @@ class KeyboardDetails
 
     /**
      * render_keyboard_details - display keyboard download boxes and details
-     * @param $id - keyboard ID
+     * @param $id - keyboard package ID
```

Comment:
A chore:
Would `/\$id/\$packageId/` help clear up ambiguity?

Reply:
We have to check each usage so we couldn't just search/replace...